Tallies/TallyClerks/collisionClerk_class.f90

```diff
@@ -194,29 +196,25 @@ subroutine reportInColl(self, p, xsData, mem, virtual)
     logical(defBool), intent(in)          :: virtual
     type(particleState)                   :: state
     integer(shortInt)                     :: binIdx, i
+    type(neutronMacroXSs)                 :: xss
+    class(neutronMaterial), pointer       :: mat
     integer(longInt)                      :: adrr
     real(defReal)                         :: scoreVal, flux
     character(100), parameter :: Here = 'reportInColl (collisionClerk_class.f90)'
 
     ! Return if collision is virtual but virtual collision handling is off
-    if (self % virtual) then
-      ! Retrieve tracking cross section from cache
-      flux = p % w / xsData % getTrackingXS(p, p % matIdx(), TRACKING_XS)
-    else
-      if (virtual) return
-      flux = p % w / xsData % getTotalMatXS(p, p % matIdx())
-    end if
+    if ((.not. self % virtual) .and. virtual) return
```

Comment:
Maybe change the property name? This is slightly confusing to read. Perhaps handleVirtual?